noo_dev_ocp.md

```diff
@@ -0,0 +1,55 @@
+# Developing network Operator (on OCP)
+
+The following instructions are useful for local speedy development of the operator.
+
+## prerequisites
+
+1. Local linux machine
+2. OCP cluster
+
+## Deployment process
+
+1. Connect to the OCP cluster using `oc login ...`
+2. execute `kubectl create namespace network-observability` - this creates the `network-observability` namespace
+3. execute `kubectl config set-context --current --namespace network-observability` - this changes the `kubectl` context to `network-observability` namespace
+4. Execute `./hack/deploy-loki.sh` - this  deploys a simple loki instance into the cluster
+4. Execute `./hack/deploy-grafana.sh` - this deploys a simple grafana instance into the cluster (user: admin password: admin)
+5. Execute `oc patch console.operator.openshift.io cluster --type='json' -p '[{"op": "add", "path": "/spec/plugins", "value": ["network-observability-plugin"]}]'` - this enables the console plugin
```

Comment:
It works but a bit strange to run before the plugin being deployed ?

I usually do that at the end. Maybe just add a note about that is enough

Reply:
done